### ISSUE TITLE
Add PDF files to uploadable media types (including commentaries)

### ISSUE THREAD

**Issue by Ironman28** [plumi/criticalcommons.content#26]
Would it be possible for Critical Commons to handle PDF files? I am asked about this regularly by people who are using CC to embed media into Scalar. How hard would it be to implement?

**Comment by ahelme** [comment]
Steve - I just added a PDF to my folder, by visiting my folder
http://www.criticalcommons.org/Members/anna/

choosing Add New > File from the green bar.

http://www.criticalcommons.org/Members/anna/test-pdf/view

Is this sufficient? I don't remember exactly how Scalar uses files, and if Scalar can access files such as the one I just added above...

**Comment by Ironman28** [comment]
I have confirmed with the Scalar team that this system will NOT work for importing PDFs into Scalar. For this to work, a PDF would have to be able to be uploaded and displayed the same way we currently handle images - including the need for a critical commentary before it was visible. If this is too difficult, please leave it for a later development cycle. If it is _not_ too difficult, this is a feature that is often asked for!

**Comment by ahelme** [comment]
Hmm, I'm a bit confused, how does a user currently add an image? 

Surely not by clicking Upload, like a video?

**Comment by Ironman28** [comment]
Yes, it works just like uploading a video. UnWeb added that functionality a couple of years ago. You can also upload audio that way.

**Comment by ahelme** [comment]
Ah, I'd forgotten that! Hmm, it sounds like also perhaps some UI work could be done to make that more apparent to the user - as it simply refers to "videos" and "clips" and what not?

For now, I guess it's a matter of adding the file type BUT I'm not sure if that might have any other implications?

**Comment by Ironman28** [comment]
Ugh. Yes, I decided not to comb through the site looking for instances that say "video" or "clip" -- that is for the future. But for now, if it's possible for PDFs to simply function (and embed?) like images, that would be great. If not, we will survive!
